circle.yml

```diff
@@ -30,7 +30,12 @@ dependencies:
 test:
   override:
     - docker run "cockroachdb/cockroach-dev" test
-    - make acceptance
+      # Kill off the previous images since we don't care for them any more,
+      # but we do want to save the logs for the following ones.
+      # `docker rm` actually errors on CircleCI but still works, hence the ';'.
```

Comment:
The usual approach to this is `|| true` so you can still put things on multiple lines:

    - docker rm $(docker ps -aq) || true
    - make acceptance